addons/mail/static/src/core/common/message_actions.js

```diff
@@ -97,6 +97,13 @@ messageActionsRegistry
         title: (component) => (component.state.showTranslation ? _t("Revert") : _t("Translate")),
         onClick: (component) => component.onClickToggleTranslation(),
         sequence: 100,
+    })
+    .add("copy", {
```

Comment:
Rename to `copy-link`. These plan to have a button to copy content in mobile soonish, so prefer be more explicit.